src/analysisd/analysisd.c

```diff
@@ -2118,7 +2092,7 @@ void * w_process_event_thread(__attribute__((unused)) void * id){
                     }
 
                     if (do_ar && execdq >= 0) {
```

Comment:
This line avoids the successful reconnection between `execd` and `analysisd`. If the connexion fails the first time, the variable `execdq` will be lower than 0 and it will never try to reconnect again. 

```suggestion
                    if (do_ar) {
```